code/src/components/TodoList.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-return-assign */
+/* eslint-disable linebreak-style */
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+
+import todos from '../reducers/todos'
+import './todoList.css'
+
+const TodoList = () => {
+  const items = useSelector((store) => store.todos.items)
+
+  let count = 0
+  const dispatch = useDispatch()
+
+  const onClickAllDone = () => {
+    dispatch(todos.actions.allDone())
+  }
+
+  return (
+    <div>
+      {items.map((todo) => (
+        <div key={todo.id} className="todo-items">
+          <div className="todo-item">
+            <input
+              className="checkbox"
+              type="checkbox"
+              checked={todo.isComplete}
+
+              onChange={() => dispatch(todos.actions.toggleComplete(todo.id))} />
+            <p className="task-description">{todo.description}</p>
+
+            <button type="button" className="btn-delete" onClick={() => dispatch(todos.actions.removeTodo(todo.id))}>
+              <span className="trash-item">🗑</span>
+            </button>
+
+            <div className="counter">
+              {todo.isComplete ? count : count += 1}
+            </div>
+
+          </div>
+
+          <div className="time">
+                        task created: {todo.timeCreated}
+          </div>
+
```

Comment:
It is no need to use so many empty lines. Put them when you need to separate different logic blocks.